10:59:39.378 com.ibm.CORBA.iiop.ClientDelegate@2e7b423f invoke:812 Thread-2868 ORBRas[default] Received SystemException  org.omg.CORBA.OBJECT_NOT_EXIST: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.OBJECT_NOT_EXIST: SERVANT_NOT_FOUND (1) for 0x4a4d4249000000100c82b1e20000000000000000000000000000000000000024000000080000000700000000  vmcid: IBM  minor code: C11  completed: No
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.getServantForRequest(ServerDelegate.java:405)
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.dispatch(ServerDelegate.java:459)
	>>	 at com.ibm.rmi.iiop.ORB.process(ORB.java:523)
	>>	 at com.ibm.CORBA.iiop.ORB.process(ORB.java:1575)
	>>	 at com.ibm.rmi.iiop.Connection.doRequestWork(Connection.java:2992)
	>>	 at com.ibm.rmi.iiop.Connection.doWork(Connection.java:2875)
	>>	 at com.ibm.rmi.iiop.WorkUnitImpl.doWork(WorkUnitImpl.java:64)
	>>	 at com.ibm.ejs.oa.pool.PooledThread.run(ThreadPool.java:118)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: IBM  minor code: C11  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.sun.jmx.remote.protocol.iiop.ProxyStub._invoke(Unknown Source)
	at org.omg.stub.javax.management.remote.rmi._RMIConnection_Stub.queryNames(Unknown Source)
	at javax.management.remote.rmi.RMIConnector$RemoteMBeanServerConnection.queryNames(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.getAdminServiceObjectName(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.isAlive(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient$ContactThread.run(Unknown Source)
, p1=<null> 
10:59:39.381 com.ibm.CORBA.iiop.ClientDelegate@b600fbf2 invoke:812 Thread-2868 ORBRas[default] Received SystemException  org.omg.CORBA.OBJECT_NOT_EXIST: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.OBJECT_NOT_EXIST: SERVANT_NOT_FOUND (1) for 0x4a4d4249000000100c82b1e20000000000000000000000000000000000000024000000080000000600000000  vmcid: IBM  minor code: C11  completed: No
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.getServantForRequest(ServerDelegate.java:405)
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.dispatch(ServerDelegate.java:459)
	>>	 at com.ibm.rmi.iiop.ORB.process(ORB.java:523)
	>>	 at com.ibm.CORBA.iiop.ORB.process(ORB.java:1575)
	>>	 at com.ibm.rmi.iiop.Connection.doRequestWork(Connection.java:2992)
	>>	 at com.ibm.rmi.iiop.Connection.doWork(Connection.java:2875)
	>>	 at com.ibm.rmi.iiop.WorkUnitImpl.doWork(WorkUnitImpl.java:64)
	>>	 at com.ibm.ejs.oa.pool.PooledThread.run(ThreadPool.java:118)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: IBM  minor code: C11  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at org.omg.stub.javax.management.remote.rmi._RMIServer_Stub.newClient(Unknown Source)
	at javax.management.remote.rmi.RMIConnector$RMIClientCommunicatorAdmin.doStart(Unknown Source)
	at com.sun.jmx.remote.internal.ClientCommunicatorAdmin.restart(Unknown Source)
	at com.sun.jmx.remote.internal.ClientCommunicatorAdmin.gotIOException(Unknown Source)
	at javax.management.remote.rmi.RMIConnector$RMIClientCommunicatorAdmin.gotIOException(Unknown Source)
	at javax.management.remote.rmi.RMIConnector$RemoteMBeanServerConnection.queryNames(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.getAdminServiceObjectName(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.isAlive(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient$ContactThread.run(Unknown Source)
, p1=<null> 
10:59:39.383 com.ibm.CORBA.iiop.ClientDelegate@2e7b423f invoke:812 Thread-2868 ORBRas[default] Received SystemException  org.omg.CORBA.OBJECT_NOT_EXIST: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.OBJECT_NOT_EXIST: SERVANT_NOT_FOUND (1) for 0x4a4d4249000000100c82b1e20000000000000000000000000000000000000024000000080000000700000000  vmcid: IBM  minor code: C11  completed: No
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.getServantForRequest(ServerDelegate.java:405)
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.dispatch(ServerDelegate.java:459)
	>>	 at com.ibm.rmi.iiop.ORB.process(ORB.java:523)
	>>	 at com.ibm.CORBA.iiop.ORB.process(ORB.java:1575)
	>>	 at com.ibm.rmi.iiop.Connection.doRequestWork(Connection.java:2992)
	>>	 at com.ibm.rmi.iiop.Connection.doWork(Connection.java:2875)
	>>	 at com.ibm.rmi.iiop.WorkUnitImpl.doWork(WorkUnitImpl.java:64)
	>>	 at com.ibm.ejs.oa.pool.PooledThread.run(ThreadPool.java:118)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: IBM  minor code: C11  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.sun.jmx.remote.protocol.iiop.ProxyStub._invoke(Unknown Source)
	at org.omg.stub.javax.management.remote.rmi._RMIConnection_Stub.close(Unknown Source)
	at javax.management.remote.rmi.RMIConnector.close(Unknown Source)
	at javax.management.remote.rmi.RMIConnector.close(Unknown Source)
	at javax.management.remote.rmi.RMIConnector$RMIClientCommunicatorAdmin.doStop(Unknown Source)
	at com.sun.jmx.remote.internal.ClientCommunicatorAdmin.restart(Unknown Source)
	at com.sun.jmx.remote.internal.ClientCommunicatorAdmin.gotIOException(Unknown Source)
	at javax.management.remote.rmi.RMIConnector$RMIClientCommunicatorAdmin.gotIOException(Unknown Source)
	at javax.management.remote.rmi.RMIConnector$RemoteMBeanServerConnection.queryNames(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.getAdminServiceObjectName(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.isAlive(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient$ContactThread.run(Unknown Source)
, p1=<null> 
10:59:39.392 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Thread-2868 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.handleRequestForQuiesceRequestedMode(OrbWorkloadRegulator.java:466)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.handleRequest(OrbWorkloadRegulator.java:350)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRequestInterceptor.receive_request(OrbWorkloadRequestInterceptor.java:47)
	>>	 at com.ibm.rmi.pi.InterceptorManager.invokeInterceptor(InterceptorManager.java:624)
	>>	 at com.ibm.rmi.pi.InterceptorManager.iterateServerInterceptors(InterceptorManager.java:528)
	>>	 at com.ibm.rmi.pi.InterceptorManager.iterateReceiveRequest(InterceptorManager.java:770)
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.dispatchInvokeHandler(ServerDelegate.java:658)
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.dispatch(ServerDelegate.java:523)
	>>	 at com.ibm.rmi.iiop.ORB.process(ORB.java:523)
	>>	 at com.ibm.CORBA.iiop.ORB.process(ORB.java:1575)
	>>	 at com.ibm.rmi.iiop.Connection.doRequestWork(Connection.java:2992)
	>>	 at com.ibm.rmi.iiop.Connection.doWork(Connection.java:2875)
	>>	 at com.ibm.rmi.iiop.WorkUnitImpl.doWork(WorkUnitImpl.java:64)
	>>	 at com.ibm.ejs.oa.pool.PooledThread.run(ThreadPool.java:118)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.getAdminServiceObjectName(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.isAlive(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient$ContactThread.run(Unknown Source)
, p1=<null> 
10:59:44.419 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 WAS Server state monitor ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.handleRequestForQuiesceRequestedMode(OrbWorkloadRegulator.java:466)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.handleRequest(OrbWorkloadRegulator.java:350)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRequestInterceptor.receive_request(OrbWorkloadRequestInterceptor.java:47)
	>>	 at com.ibm.rmi.pi.InterceptorManager.invokeInterceptor(InterceptorManager.java:624)
	>>	 at com.ibm.rmi.pi.InterceptorManager.iterateServerInterceptors(InterceptorManager.java:528)
	>>	 at com.ibm.rmi.pi.InterceptorManager.iterateReceiveRequest(InterceptorManager.java:770)
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.dispatchInvokeHandler(ServerDelegate.java:658)
	>>	 at com.ibm.CORBA.iiop.ServerDelegate.dispatch(ServerDelegate.java:523)
	>>	 at com.ibm.rmi.iiop.ORB.process(ORB.java:523)
	>>	 at com.ibm.CORBA.iiop.ORB.process(ORB.java:1575)
	>>	 at com.ibm.rmi.iiop.Connection.doRequestWork(Connection.java:2992)
	>>	 at com.ibm.rmi.iiop.Connection.doWork(Connection.java:2875)
	>>	 at com.ibm.rmi.iiop.WorkUnitImpl.doWork(WorkUnitImpl.java:64)
	>>	 at com.ibm.ejs.oa.pool.PooledThread.run(ThreadPool.java:118)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.isAlive(Unknown Source)
	at com.ibm.ws.management.AdminClientImpl.isAlive(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection.isConnectionAlive(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour.getWebSphereJmxConnection(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour.getJmxConnection(Unknown Source)
	at com.ibm.ws.ast.st.common.core.internal.AbstractWASServerBehaviour.removeApplicationNotificationListener(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour.disconnectWebSphereJmxConnection(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour$ServerStateMonitor.handleServerStop(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour$ServerStateMonitor.run(Unknown Source)
, p1=<null> 
10:59:49.967 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:50.637 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:51.529 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:52.155 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 WAS Server state monitor ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.getServerMBean(Unknown Source)
	at com.ibm.ws.management.AdminClientImpl.getServerMBean(Unknown Source)
	at com.ibm.websphere.management.configservice.ConfigServiceProxy.<init>(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection.getConfigServiceProxy(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour.getWebSphereJmxConnection(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour.getJmxConnection(Unknown Source)
	at com.ibm.ws.ast.st.common.core.internal.AbstractWASServerBehaviour.removeApplicationNotificationListener(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour.disconnectWebSphereJmxConnection(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour$ServerStateMonitor.handleServerStop(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour$ServerStateMonitor.run(Unknown Source)
, p1=<null> 
10:59:52.160 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 WAS Server state monitor ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.getAdminServiceObjectName(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.resetFilter(Unknown Source)
	at com.ibm.ws.management.event.RemoteReceiver.resubscribe(Unknown Source)
	at com.ibm.ws.management.event.RemoteReceiver.setFilter(Unknown Source)
	at com.ibm.ws.management.event.ClientNotificationService.updateRemoteFilter(Unknown Source)
	at com.ibm.ws.management.event.ClientNotificationService.removeNotificationListenerExtended(Unknown Source)
	at com.ibm.ws.management.AdminClientImpl.removeNotificationListenerExtended(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection.removeApplicationtNotificationListener(Unknown Source)
	at com.ibm.ws.ast.st.common.core.internal.AbstractWASServerBehaviour.removeApplicationNotificationListener(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour.disconnectWebSphereJmxConnection(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour$ServerStateMonitor.handleServerStop(Unknown Source)
	at com.ibm.ws.ast.st.v8.core.internal.WASServerBehaviour$ServerStateMonitor.run(Unknown Source)
, p1=<null> 
10:59:52.534 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:53.537 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:54.541 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:55.546 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:56.549 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:57.172 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 WAS communication ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.isAlive(Unknown Source)
	at com.ibm.ws.management.AdminClientImpl.isAlive(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClientPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.access$000(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory$1.run(Unknown Source)
	at com.ibm.ws.security.util.AccessController.doPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection.connectAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection$AgentThread.run(Unknown Source)
, p1=<null> 
10:59:57.179 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 WAS communication ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.<init>(Unknown Source)
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClientPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.access$000(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory$1.run(Unknown Source)
	at com.ibm.ws.security.util.AccessController.doPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection.connectAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection$AgentThread.run(Unknown Source)
, p1=<null> 
10:59:57.183 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 WAS communication ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.<init>(Unknown Source)
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClientPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.access$000(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory$1.run(Unknown Source)
	at com.ibm.ws.security.util.AccessController.doPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection.connectAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection$AgentThread.run(Unknown Source)
, p1=<null> 
10:59:57.554 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:58.557 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
10:59:59.561 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
11:00:00.565 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
11:00:01.569 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
11:00:02.471 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 WAS communication ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.<init>(Unknown Source)
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClientPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.access$000(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory$1.run(Unknown Source)
	at com.ibm.ws.security.util.AccessController.doPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection.connectAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection$AgentThread.run(Unknown Source)
, p1=<null> 
11:00:02.477 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 WAS communication ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.<init>(Unknown Source)
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClientPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.access$000(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory$1.run(Unknown Source)
	at com.ibm.ws.security.util.AccessController.doPrivileged(Unknown Source)
	at com.ibm.websphere.management.AdminClientFactory.createAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection.connectAdminClient(Unknown Source)
	at com.ibm.ws.ast.st.jmx.core.internal.WebSphereGenericJmxConnection$AgentThread.run(Unknown Source)
, p1=<null> 
11:00:02.574 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
11:00:03.579 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
11:00:04.582 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
11:00:05.585 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
11:00:06.591 com.ibm.CORBA.iiop.ClientDelegate@7fc3b3f3 invoke:812 Non-deferrable Alarm : 0 ORBRas[default] Received SystemException  org.omg.CORBA.COMM_FAILURE: 
	>> SERVER (id=c82b1e2, host=M6600) TRACE START:
	>>    org.omg.CORBA.COMM_FAILURE:   vmcid: 0x49421000  minor code: 40  completed: No
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.getException(OrbWorkloadRegulator.java:816)
	>>	 at com.ibm.ws.runtime.workloadcontroller.OrbWorkloadRegulator.quiesceComplete(OrbWorkloadRegulator.java:792)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesceCompleteWorkloads(WorkloadController.java:428)
	>>	 at com.ibm.ws.runtime.workloadcontroller.WorkloadController.quiesce(WorkloadController.java:216)
	>>	 at com.ibm.ws.runtime.component.WLCImpl.stop(WLCImpl.java:144)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponent(ContainerHelper.java:476)
	>>	 at com.ibm.ws.runtime.component.ContainerHelper.stopComponents(ContainerHelper.java:460)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stopComponents(ContainerImpl.java:650)
	>>	 at com.ibm.ws.runtime.component.ContainerImpl.stop(ContainerImpl.java:638)
	>>	 at com.ibm.ws.runtime.component.ServerImpl.stop(ServerImpl.java:615)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook$1.run(ServerCollaborator.java:860)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAs(ContextManagerImpl.java:5445)
	>>	 at com.ibm.ws.security.auth.ContextManagerImpl.runAsSystem(ContextManagerImpl.java:5571)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$ShutdownHook.run(ServerCollaborator.java:850)
	>>	 at com.ibm.ws.runtime.component.ServerCollaborator$StopAction.alarm(ServerCollaborator.java:809)
	>>	 at com.ibm.ejs.util.am._Alarm.run(_Alarm.java:133)
	>>	 at com.ibm.ws.util.ThreadPool$Worker.run(ThreadPool.java:1659)
	>> SERVER (id=c82b1e2, host=M6600) TRACE END.
  vmcid: 0x49421000  minor code: 40  completed: No
	at sun.reflect.GeneratedConstructorAccessor19.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage._getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ReplyMessage.getSystemException(Unknown Source)
	at com.ibm.rmi.iiop.ClientResponseImpl.getSystemException(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.intercept(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.rmi.corba.ClientDelegate.invoke(Unknown Source)
	at com.ibm.CORBA.iiop.ClientDelegate.invoke(Unknown Source)
	at org.omg.CORBA.portable.ObjectImpl._invoke(Unknown Source)
	at com.ibm.WsnOptimizedNaming._NamingContextStub.resolve_complete_info(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl$2.run(Unknown Source)
	at com.ibm.ws.naming.util.CommonHelpers.retry(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.cosResolve(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.doLookup(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookupExt(Unknown Source)
	at com.ibm.ws.naming.jndicos.CNContextImpl.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at com.ibm.ws.naming.util.WsnInitCtx.lookup(Unknown Source)
	at javax.naming.InitialContext.lookup(Unknown Source)
	at com.ibm.websphere.management.remote.WsProvider.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.newJMXConnector(Unknown Source)
	at javax.management.remote.JMXConnectorFactory.connect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.reconnect(Unknown Source)
	at com.ibm.ws.management.connector.jsr160rmi.JSR160RMIConnectorClient.pullNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.getNotifications(Unknown Source)
	at com.ibm.ws.management.event.PullRemoteReceiver.alarm(Unknown Source)
	at com.ibm.ejs.util.am._Alarm.run(Unknown Source)
	at com.ibm.ws.util.ThreadPool$Worker.run(Unknown Source)
, p1=<null> 
